source/fem-dem/vans_assemblers.cc

```diff
@@ -1179,26 +1178,27 @@ GLSVansAssemblerBeetstra<dim>::calculate_particle_fluid_interactions(
       superficial_velocity =
         relative_velocity[particle_number] * cell_void_fraction;
 
-      // Beetstra normalized frag force
-      normalized_drag_force =
-        10 * (1 - cell_void_fraction) / (pow(cell_void_fraction, 2)) +
-        pow(cell_void_fraction, 2) *
-          (1 + 1.5 * sqrt((1 - cell_void_fraction))) +
-        0.413 * Re_p[particle_number] / (24 * pow(cell_void_fraction, 2)) *
-          ((1 / cell_void_fraction) +
-           3 * (1 - cell_void_fraction) * cell_void_fraction +
-           8.4 * pow(Re_p[particle_number], -0.343)) /
-          (1 + pow(10, 3 * (1 - cell_void_fraction)) *
-                 pow(Re_p[particle_number],
-                     -(1 + 4 * (1 - cell_void_fraction)) * 0.5));
-
-      drag_force = normalized_drag_force * 3 * M_PI * viscosity * density *
-                   particle_properties[DEM::PropertiesIndex::dp] *
-                   superficial_velocity;
-
-      beta_drag +=
-        drag_force.norm() /
-        (density * (relative_velocity[particle_number].norm()) + 1e-15);
+      // Beetstra drag coefficient
```

Comment:
Since you are reframing the drag model, the comment in the .h file needs to be changed as well. I would add the reference in the .h and put a note on this at the documentation.

Reply:
Done